rand_core/src/lib.rs

```diff
@@ -343,11 +343,13 @@ pub trait SeedableRng: Sized {
 
 impl<'a, R: RngCore + ?Sized> RngCore for &'a mut R {
     #[inline(always)]
+    #[cfg_attr(feature = "cargo-clippy", allow(inline_always))]
```

Comment:
No, I don't want these extra lints here; if we go down that road we end up with a mess (been there). Just use `-A inline_always`.